sg_jira/shotgun_session.py

```diff
@@ -252,12 +252,12 @@ def match_entity_by_name(self, name, entity_types, shotgun_project):
         """
         for entity_type in entity_types:
             name_field = self.get_entity_name_field(entity_type)
-            filter = [[name_field, "is", name]]
+            filters = [[name_field, "is", name]]
             fields = [name_field]
             if self.is_project_entity(entity_type):
-                filter.append(["project", "is", shotgun_project])
+                filters.append(["project", "is", shotgun_project])
                 fields.append("project")
-            sg_value = self.find_one(entity_type, filter, fields,)
+            sg_value = self.find_one(entity_type, filters, fields,)
```

Comment:
Not technically required, but it's bad practice to use built in Python keywords as variables so I updated it.